<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>679</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="GraphicsView" name="leftGraphicsView"/>
     </item>
     <item>
      <widget class="GraphicsView" name="rightGraphicsView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>28</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_AS"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTransform">
    <property name="title">
     <string>转换</string>
    </property>
    <addaction name="actionAddGaussNoise"/>
    <addaction name="action_ArithmeticMeanFilter"/>
    <addaction name="actionGeometricMeanFilter"/>
    <addaction name="actionAdaptiveFilter"/>
    <addaction name="actionTransGray"/>
    <addaction name="actionAddSaltNoise"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTransform"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSave_AS">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdjust">
   <property name="text">
    <string>Adjust</string>
   </property>
  </action>
  <action name="actionAddGaussNoise">
   <property name="text">
    <string>AddGaussNoise</string>
   </property>
  </action>
  <action name="action_ArithmeticMeanFilter">
   <property name="text">
    <string> ArithmeticMeanFilter</string>
   </property>
  </action>
  <action name="actionGeometricMeanFilter">
   <property name="text">
    <string>GeometricMeanFilter </string>
   </property>
  </action>
  <action name="actionAdaptiveFilter">
   <property name="text">
    <string>AdaptiveFilter</string>
   </property>
  </action>
  <action name="actionTransGray">
   <property name="text">
    <string>TransGray</string>
   </property>
  </action>
  <action name="actionAddSaltNoise">
   <property name="text">
    <string>AddSaltNoise</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
